### ISSUE TITLE
Android libandroid_tensorflow_inference - "No OpKernel was registered to support Op 'FloorMod' with these attrs" when using tf.nn.atrous_conv2d

### ISSUE THREAD

**Issue by telefire** [tensorflow/tensorflow#9763]
hi, everyone,

We are trying to load a pb file inside Android. The pb file is generated with python 2.7.0 and Tensorflow 1.0.1.   We have this following error:

```
exception triggered java.lang.IllegalArgumentException: No OpKernel was registered to support Op 'FloorMod' with these attrs.  Registered devices: [CPU], Registered kernels:
                                                                    <no registered kernels>                                                                  
                                                                  	 [[Node: WCon/atrous_conv2d/mod = FloorMod[T=DT_INT32, _device="/device:CPU:0"](WCon/atrous_conv2d/add_1, WCon/atrous_conv2d/mod/y)]]
```

The network definition is using tf.nn.atrous_conv2d(i, k, dilation_rate, padding=padding)

And our freezing graph code is also pretty standard.

We have no problem running this pb files for inference with python under Tensorflow 1.0.1

But when running on Android phone, we will see the above error, I tried several nightly build TF Android Inference library including the latest from https://ci.tensorflow.org/view/Nightly/job/nightly-android/    The result is the same

My feeling is that the inference on Tensorflow python and Tensorflow Android has a huge gap?

Thanks for any ideas!

**Comment by girving** [comment]
Duplicate of https://github.com/tensorflow/tensorflow/issues/9476.

**Comment by pavelchristof** [comment]
This is still an issue for the TensorFlow AAR on maven ('org.tensorflow:tensorflow-android:1.3.0'), which makes it kind of.. useless.
